frontend/src/views/camp/Dashboard.vue

```diff
@@ -330,36 +350,69 @@ export default {
   async mounted() {
     this.api.reload(this.camp())
 
-    const [loggedInUser] = await Promise.all([
-      this.$auth.loadUser(),
-      this.camp().periods()._meta.load,
+    const loadingDataPromise = Promise.all([
+      this.loggedInUser._meta.load,
       this.camp().activities()._meta.load,
-      this.camp().categories()._meta.load,
     ])
 
-    this.loggedInUser = loggedInUser
+    // Once camp data is loaded validate and map values from url param to filter
+    await Promise.all([
```

Comment:
Is there a specific reason we need 2 separate calls for `Promise.all`? Or could we put all the loading promises into one `Promise.all` and await this one?

Reply:
Nope, that's leftover code from when I tried some performance optimisation.